### PR TITLE
Shared Theme and Storybooks

### PR DESCRIPTION
**Pull request by chriskrogh**
// `yarn.lock diffs r big`

## What's changed

Our theme has been completely moved to the shared package. This is because we have some components in there that need the theme. I'm also thinking that we'll also most likely be sharing this theme with `marketplace`?

I also set up `storybook` in shared with the same plugins that we use in sc.

## Notes
Next I'll be adding the shared component `OrderDetails` that all of this set up is ACTUALLY for😂

### CODE REVIEW

**Comment by jtaaa** [comment]
Task linked: [CU-bcgt7f Add OrderDetails](https://app.clickup.com/t/bcgt7f)

File: packages/shared/.storybook/preview.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```
Comment (jtaaa) [inline packages/shared/.storybook/preview.js:2]: cool!

**Review by jtaaa — COMMENTED** [review]
LGTM just some newlines at EOF are missing. Our eslint setup probably isn't linting JS

**Comment by chriskrogh** [comment]
updated eof spacing in 0462245105426281beaa770647fcbc8e73a5e242

**Comment by jtaaa** [comment]
darnit it didn't close the PR

**Pull request closed without merging**